.vsts-ci.yml

```diff
@@ -20,11 +20,11 @@ phases:
     displayName: Start collect diagnostics
     condition: and(succeeded(), eq(variables.collect_diagnostics, 'true'))
 
-  # Use node 6, npm 3
-  - task: NodeTool@0
-    displayName: Use node 6.10.3
-    inputs:
-      versionSpec: "6.10.3"
+  # Use node 8, npm 5		
+  - task: NodeTool@0		
+    displayName: Use node 8.9.1		
```

Comment:
latest node 8 would be fine here. the reason we need specific versions for node 6 and 5 is because those specific version ship with the agent